slack-lint/src/test/java/slack/lint/resources/WrongResourceImportAliasDetectorTest.kt

```diff
@@ -0,0 +1,270 @@
+/*
+ * Copyright (C) 2022 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package slack.lint.resources
+
+import com.android.tools.lint.detector.api.Detector
+import org.junit.Test
+import slack.lint.BaseSlackLintTest
+
+class WrongResourceImportAliasDetectorTest : BaseSlackLintTest() {
+
+  override fun getDetector(): Detector = WrongResourceImportAliasDetector()
+
+  override fun getIssues() = listOf(WrongResourceImportAliasDetector.ISSUE)
+
+  @Test
+  fun `test success`() {
+    lint()
+      .files(
+        kotlin(
+          """
+          package slack.pkg.subpackage
+
+          import slack.l10n.R as L10nR
+          import slack.pkg.R
+
+           class MyClass {
+
+             init {
+                  val appName = getString(L10R.string.app_name)
+              }
+
+           }
+          """
+        )
+          .indented()
+      )
+      .run()
+      .expectClean()
+  }
+
+  @Test
+  fun `test failure no references`() {
+    lint()
+      .files(
+        kotlin(
+          """
+          package slack.pkg.subpackage
+
+          import slack.l10n.R as L10R
+
+           class MyClass
+          """
+        )
+          .indented()
+      )
+      .run()
+      .expect(
+        """
+        src/slack/pkg/subpackage/MyClass.kt:3: Error: Please use L10nR as an import alias here [WrongResourceImportAlias]
+        import slack.l10n.R as L10R
+        ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        1 errors, 0 warnings
+        """.trimIndent()
+      )
+      .expectFixDiffs(
+        """
+        Autofix for src/slack/pkg/subpackage/MyClass.kt line 3: Replace import alias:
+        @@ -3 +3
+        - import slack.l10n.R as L10R
+        + import slack.l10n.R as L10nR
+        """.trimIndent()
+      )
+  }
+
+  @Test
+  fun `test failure one reference`() {
+    lint()
+      .files(
+        kotlin(
+          """
+          package slack.pkg.subpackage
+
+          import slack.uikit.resources.R as SlackKitR
+          import slack.l10n.R as L10R
+
+           class MyClass {
```

Comment:
seeing this in a few test sources, please fix the others too

```suggestion
          class MyClass {
```

Reply:
e71cdd218da1d1bdb6b5684474694302bc7fc973